### ISSUE TITLE
Allow None value to be serialized as null

### ISSUE THREAD

**Issue by pilb** [antidot/Pyckson#13]
Hello,

Could we have a feature that serialize a None attribute to null like below?

```
def test_class_with_optional_attribute(self):
       @pyckson
        class Foo:
            def __init__(self, a: int, b: str = None):
                self.a = a
                self.b = b

        result = serialize(Foo(42))

        self.assertEqual(result, {'a': 42, 'b': null})
```

Thank you,

Philippe

**Comment by JGiard** [comment]
sorry, i had forgotten about this issue, it will probably in the form of a configuration option like what I've done for dates : https://pyckson.readthedocs.io/en/latest/advanced.html#custom-date-formatters

**Comment by JGiard** [comment]
this should be available in version 1.7.0
https://pyckson.readthedocs.io/en/latest/advanced.html#nulls-serialisation

**Comment by pilb** [comment]
Thank you @JGiard :)
